content/compose/release-notes.md

```diff
@@ -11,6 +11,35 @@ aliases:
 
 For more detailed information, see the [release notes in the Compose repo](https://github.com/docker/compose/releases/).
 
+## 2.24.0
+{{< release-date date="2024-01-11" >}}
+
+### Update
+- Dependencies upgrade: bump docker cli v1.25 beta1 
+- Dependencies upgrade: bump cli to 25.0.0-beta.3
+- Dependencies upgrade: bump compose-go to 2.0.0-beta.3
+- Dependencies upgrade: bump ddev to 1.22.4 
+- Dependencies upgrade: bump golang to 1.21.6 
+- Dependencies upgrade: bump golangci-lint to 1.55.2
```

Comment:
I'm not sure we need to mention the dependency bump of `ddev` and `golang-ci`, those are more linked to CI
```suggestion
- Dependencies upgrade: bump docker cli v1.25 beta1 
- Dependencies upgrade: bump cli to 25.0.0-beta.3
- Dependencies upgrade: bump compose-go to 2.0.0-beta.3
- Dependencies upgrade: bump golang to 1.21.6 
```

Reply:
removed in https://github.com/docker/docs/pull/19089/commits/4bdc7942d3aa6d5a8e9cc18c6019098908d09472